code/controllers/subsystem/processing/SSstation.dm

```diff
@@ -0,0 +1,74 @@
+PROCESSING_SUBSYSTEM_DEF(station)
+	name = "Station"
+	init_order = INIT_ORDER_STATION
+	flags = SS_TICKER
+	wait = 5 SECONDS
+	cpu_display = SS_CPUDISPLAY_DEFAULT
+	offline_implications = "Station events will no longer process. No intervention needed at this time."
+
+	///A list of currently active station traits
+	var/list/station_traits = list()
+	///Assoc list of trait type || assoc list of traits with weighted value. Used for picking traits from a specific category.
+	var/list/selectable_traits_by_types = list(STATION_TRAIT_POSITIVE = list(), STATION_TRAIT_NEUTRAL = list(), STATION_TRAIT_NEGATIVE = list())
+
+/datum/controller/subsystem/processing/station/Initialize()
+	SetupTraits()
+
+///Rolls for the amount of traits and adds them to the traits list
+/datum/controller/subsystem/processing/station/proc/SetupTraits()
+
+	if(fexists("data/next_traits.txt"))
+		var/forced_traits_contents = file2list("data/next_traits.txt")
+		fdel("data/next_traits.txt")
+		var/list/temp_list = splittext(forced_traits_contents[1], ",") //Tomorow, you need to make delete the file before making it, then split it
+
+		for(var/trait_text_path in temp_list)
+			var/station_trait_path = text2path(trait_text_path)
+			if(!ispath(station_trait_path, /datum/station_trait) || station_trait_path == /datum/station_trait)
+				var/message = "Invalid station trait path [station_trait_path] was requested in the future station traits!"
+				log_game(message)
+				message_admins(message)
+				continue
+
+			setup_trait(station_trait_path)
+
+		return
+
+	for(var/i in subtypesof(/datum/station_trait))
+		var/datum/station_trait/trait_typepath = i
+
+		// If forced, (probably debugging), just set it up now, keep it out of the pool.
+		if(initial(trait_typepath.force))
+			setup_trait(trait_typepath)
+			continue
```

Comment:
Do we want to remove, or keep this debugging code?

Reply:
Keep in case probably